### PR TITLE
Diefi

### PR DESCRIPTION
**Pull request by mickeyjoes**

### CODE REVIEW

**Comment by mickeyjoes** [comment]
Fixed broken test in last commit

**Comment by mickeyjoes** [comment]
@zwergius  you can do it

File: src/kryptos/digest.test.js [hunk 1]
```diff
@@ -6,6 +6,6 @@ test('Test hash anything - mix of variables, objects, arrays, strings etc.', asy
   const b = [11, 22, 33]
   const c = { a: 1, b: [1, 2, 3], c: 'test' }
   const d = 123213
-  const hash = await hashAnything(a, b, c, d)
+  const hash = await hashAnything('SHA-256', a, b, c, d)
```
Comment (zwergius) [inline src/kryptos/digest.test.js:9]: SHA_256.name ?

**Review by zwergius — APPROVED** [review]
LGTM
ok usa

File: src/kryptos/digest.test.js (hunk 1 above)
Comment (mickeyjoes) [reply src/kryptos/digest.test.js:9]: yes could be, for tests hardcoded it

**Pull request merged**